src/components/layout/Seo.js

```diff
@@ -12,7 +12,14 @@ import favicon192 from "assets/favicon/android-chrome-192x192.png";
 import favicon512 from "assets/favicon/android-chrome-512x512.png";
 import StellarDocsImage from "assets/images/og_developers.jpg";
 
-export const Seo = ({ title, description = "", path }) => {
+export const Seo = ({
+  title,
+  description = "",
+  path,
+  omitPredicate = () => true,
+  meta = [],
+  link = [],
```

Comment:
Arguably overengineered. We want to omit the `<link rel="canonical">` head tag on the `/api/` page (because it's not the canonical URL for any of that content and Google is confused), but I didn't want to special-case it